metabase/metabase
# allow parametrized link cards

[Related product doc](https://www.notion.so/metabase/Support-variables-in-Iframe-and-Link-cards-15a69354c9018075955ded3076735fcd)

### Description

Supports parametrized link dashboard cards.

### How to verify

- Create a dashboard
- Add a link card with a URL like `https://example.com/{{foo}}/{{bar}}`
- Add parameters and connect them to `foo` and `bar` variables
- Change filter values and ensure it changes the link

### Demo

<img width="391" alt="Screenshot 2024-12-17 at 8 49 54 PM" src="https://github.com/user-attachments/assets/cc0c87ba-22d0-4d7e-822e-0ded9d9b34d3" />

<img width="432" alt="Screenshot 2024-12-17 at 8 49 47 PM" src="https://github.com/user-attachments/assets/2bbec700-4d1f-4085-8846-57cd4878a7f6" />

### Checklist
